frontend/src/navbar.tsx

```diff
@@ -63,9 +63,9 @@ export function Navbar(props: BoxProps) {
                   size="sm"
                   title="Toggle display for externally blocked pull requests"
                   colorScheme="blue"
-                  variant={showExtBlocked ? 'solid' : 'ghost'}
+                  variant='ghost'
                   onClick={toggleShowExtBlocked}>
-                  <FontAwesomeIcon icon={faEyeSlash}/>
+                  <FontAwesomeIcon icon={showExtBlocked ? faEyeSlash : faEye}/>
```

Comment:
What was the reasoning for this change?

From my perspective, I associate the `faEyeSlash` icon with non-QAable pulls and the `faEye` icon with pulls currently being QAed. Therefore, each icon has its own meaning. However, it now sees that the toggling will change the icon used so this may cause some confusion.

Reply:
Loading up Pulldasher before this change would have the button selected initially like this

![selected](https://user-images.githubusercontent.com/95656772/188212533-faa29611-b2d0-494f-9735-30a0e32d741d.jpg)

This change was just a quick way to determine if we are showing externally blocked pull requests or not. I didn't really think about the QA icon there since this button icon would initially be shown as the faEyeSlash and only then changed when someone wants to hide externally blocked PRs; I don't really think it could cause much confusion, but I do understand the concern.

Reply:
I'll keep this as just the faEyeSlash and then have the border appear around it if it's active (like Cryogenic Storage)

Reply:
https://github.com/iFixit/pulldasher/pull/334/commits/24c603dd10354fd8b984c81cc1e72e97c57760dd